recipes-core/images/avnet-image-full.inc

```diff
@@ -115,14 +120,9 @@ IMAGE_INSTALL:append:minized-sbc = "\
         wpa-supplicant \
 "
 
-IMAGE_INSTALL:append:mz = "\
-"
-
 IMAGE_INSTALL:append:pz = "\
-        haveged \
         pciutils \
         performance-tests \
-        util-linux-mkfs \
```

Comment:
I understand why this was removed from the minimal image that is meant to be booted from QSPI, but I think it should probably be left in avnet-image-full that will be used for SD card and eMMC boot.

Reply:
This is included in the `#IMAGE_INSTALL:append:pz` in the `avnet-image-minimal.inc` recipe which the `avnet-image-full.inc` recipe requires. Therefore, it's not necessary to have this in both places.